integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -191,45 +182,89 @@ <h3 class="heading font-bold text-black">
                             <label>
                                 {% trans "Map" %}
                             </label>
-                            <div id="no_coordinates_alert"
-                                 class="bg-blue-100 border-l-4 border-blue-500 text-blue-500 px-4 py-3 my-2 hidden"
-                                 role="alert">
-                                {% trans "Map preview will be available when coordinates are given." %}
```

Comment:
I think with the new feature, it makes sense to show the map all the time and allow users to either set the position via drag & drop or via clicking on the map. No need to hide and show the map when there's always something to do.